apps/happy-blocks/src/pricing-plans/components/billing-options.tsx

```diff
@@ -0,0 +1,42 @@
+import { createInterpolateElement } from '@wordpress/element';
+import { FunctionComponent } from 'react';
+import { PricingPlanBilling } from '../types';
+
+const Promo: FunctionComponent = ( { children } ) => (
+	<span className="wp-block-a8c-pricing-plans__detail-billings-item-promo">({ children })</span>
```

Comment:
I feel like the class name is too long. Perhaps drop the `detail-billings-item` part?